Header header
std_msgs/Float64 Quality
geometry_msgs/Point grasp_point
geometry_msgs/Point grasp_dir
std_msgs/Float64 palm_rotation
std_msgs/Float64 palm_fallback
geometry_msgs/Pose gripper_pose
geometry_msgs/Pose pre_gripper_pose